OutOfSchool/OutOfSchool.WebApi.Tests/Services/ChildServiceTests.cs

```diff
@@ -260,37 +260,81 @@ public async Task GetApprovedByWorkshopId_NoExistingChildren_ShouldReturnEmptySe
     //        Assert.ThrowsAsync<ArgumentException>(() => childService.UpdateChildForParetWithSpecifiedUserId(child));
     //    }
 
-    //    [Test]
-    //    public void ChildService_Delete_DeletesChild()
-    //    {
-    //        ChildDto child = new ChildDto { Id = 1, FirstName = "fn1", LastName = "ln1", MiddleName = "mn1", DateOfBirth = new DateTime(2003, 11, 9), Gender = Gender.Male, ParentId = 1, SocialGroupId = 2 };
-    //        mockRepository.Setup(m => m.Delete(It.IsAny<Child>()));
-    //        mockRepository.Setup(m => m.GetById(It.IsAny<long>())).Returns(Task.FromResult(GetTestChildEntities().First()));
+    [Test]
+    public async Task ChildService_DeleteWithTechAdminRole_DeletesChild()
+    {
+        // Arrange
+        var child = new Child()
+        {
+            Id = Guid.NewGuid(),
+            Parent = new Parent() { UserId = Guid.NewGuid().ToString() },
+        };
 
-    //        IChildService childService = new ChildService(mockRepository.Object, logger.Object, localizer.Object);
+        var childList = new List<Child> { child }.BuildMock();
 
-    //        childService.DeleteChildCheckingItsUserIdProperty(1);
+        childRepositoryMock.Setup(m => m.GetByFilterNoTracking(It.IsAny<Expression<Func<Child, bool>>>(), It.IsAny<string>()))
+            .Returns(childList);
 
-    //        mockRepository.Verify(x => x.Delete(It.Is<Child>(c => c.Id == child.Id)), Times.Once);
-    //    }
+        childRepositoryMock.Setup(m => m.Delete(It.IsAny<Child>())).Returns(Task.CompletedTask);
+        applicationRepositoryMock.Setup(x => x.DeleteChildApplications(It.IsAny<Guid>())).Returns(Task.CompletedTask);
```

Comment:
(Test code readability) Please avoid usage of `It.IsAny` in `mock.Setup()` when you definitely know what parameter will be send there. This will enforce your tests with knowledge that i.e. deletion should be called only with this specific `child` or 
 `child.Id` and let you wite in Assert section just simple `mock.VerifyAll()` instead of duplicated assertions for all setups previously done.

Reply:
Fixed